rita/src/rita_common/rita_loop/mod.rs

```diff
@@ -201,6 +204,88 @@ impl Handler<Tick> for RitaLoop {
                 }).then(|_| Ok(())),
         );
 
+        let full_node = get_web3_server();
+        let web3 = Web3Client::new(&full_node);
+        let our_address = SETTING.get_payment().eth_address;
+        trace!("About to make web3 requests to {}", full_node);
+        Arbiter::spawn(
```

Comment:
In this chain of unrelated futures I'd probably consider joining them together and execute in a `Arbiter::spawn` so this way once all web3 calls are done you could do a single update (instead of separate async changes to the structure).

Reply:
tried that, gets really awkward.